interactive/templates/interactive/analysis_request_output.html

```diff
@@ -11,22 +11,15 @@ <h1 class="text-center text-3xl font-extrabold text-slate-900 sm:w-full sm:max-w
 {% block template_content %}
 <div class="text-center text-slate-800">
   <div class="prose prose-oxford">
-    <h2>{{ analysis.codelist }}</h2>
-    <p>Some content goes here.</p>
-    <p>--Deciles chart--</p>
-  </div>
-</div>
-<div class="mt-12 text-slate-800">
-  <div class="prose prose-oxford">
-    <h3>Most common codes (<a href="https://www.opencodelists.org/codelist/{{ analysis.codelist }}">Codelist</a>)</h3>
-    <table>
-      <thead><th>Code</th><th>Description</th><th>Proportion of codes (%)</th></thead>
-      <tbody><td>...</td><td>...</td><td>...</td></tbody>
-    </table>
+    <h2>{{ analysis.title }}</h2>
+    <p>The aim of this OpenSAFELY-Interactive report is to describe trends and variation in clinical codes to inform further analyses. It is not intended to be a tool for “performance monitoring”.</p>
```

Comment:
"OpenSAFELY-Interactive" or "OpenSAFELY Interactive"?

Reply:
"“performance monitoring”" or "performance monitoring"?

Reply:
I copied and pasted from Brian's [mock-up](https://docs.google.com/document/d/1B-fT6YwU_Vp_pM1UK8kAcOrYkQJGROFEjsRf88HYlm4/edit), the quotes come from there (as did the extraneous hyphen). I suppose "normal" quotes look better and more consistent?